dev/com.ibm.ws.transaction/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -78,3 +78,12 @@ manual=Manual
 
 default.max.shutdown.delay=Default maximum shutdown delay
 default.max.shutdown.delay.desc=Default maximum shutdown delay
+
+lease.check.strategy=Lease check strategy
+lease.check.strategy.desc=The algorithm used by the recovery service to check the recovery lease log
```

Comment:
**change**
The algorithm used by the recovery service to check the recovery lease log
**to**
The algorithm that is used by the recovery service to check the recovery lease log

Reply:
or, change it to "The algorithm that the recovery service uses to check the recovery lease log"